tests/integration_tests/test_account.py

```diff
@@ -18,15 +18,20 @@ def test_get_transaction_count(ethermint_rpc_ws, geth):
         n0 = w3.eth.get_transaction_count(receiver, blk)
         # ensure transaction send in new block
         w3_wait_for_new_blocks(w3, 1, sleep=0.1)
+        amount = 1000
         txhash = w3.eth.send_transaction(
             {
                 "from": sender,
                 "to": receiver,
-                "value": 1000,
+                "value": amount,
             }
         )
         receipt = w3.eth.wait_for_transaction_receipt(txhash)
         assert receipt.status == 1
         [n1, n2] = [w3.eth.get_transaction_count(receiver, b) for b in [blk, "latest"]]
         assert n0 == n1
         assert n0 == n2
+
+        [b0, b1, b2] = [w3.eth.get_balance(receiver, b) for b in ["earliest", blk, "latest"]]
```

Comment:
@VictorTrustyDev I fail to reproduce earliest balance issue with [v0.20.0](https://github.com/evmos/ethermint/tree/v0.20.0) or main

Reply:
@mmsqe because cronos has block 1 but evmos doesn't. The genesis block of evmos is 58701

Sorry for late response since I was on Tet holidays and not read mails

Reply:
np, you mean should align genesis one [here](https://github.com/evmos/ethermint/blob/26a2855c6a95b3e1fab0626ae7b29f62e901252d/rpc/types/block.go#L176)?

Reply:
Yeah when "earliest", the block number was fixed to 0 which is going to be resolved to 1 in many places [like this](https://github.com/evmos/ethermint/blob/26a2855c6a95b3e1fab0626ae7b29f62e901252d/x/evm/keeper/grpc_query.go#L405-L410). Probably we should have a param config of genesis height or somehow resolve it during boot